releases/release-1.31/release-notes/maps/pr-124320-map.yaml

```diff
@@ -0,0 +1,23 @@
+pr: 124320
```

Comment:
Let's delete this map file- it's a known krel bug: https://github.com/kubernetes/release/issues/3516